source/_integrations/fully_kiosk.markdown

```diff
@@ -64,6 +66,10 @@ The following controls are available:
 - Play and stop media files
 - Set device volume
 
+The following is available as cameras:
```

Comment:
I'm not sure I understand this. what is meant by "available as cameras"?

Reply:
Reworded it a bit.